app/Models/Categories.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Categories extends Model
```

Comment:
A model representa um registro do banco de dados. Cada instância da model é único. Dessa forma, tu sempre precisa nomear as models no singular
```suggestion
class Category extends Model
```